ddtrace/opentracer/span_context.py

```diff
@@ -1,15 +1,31 @@
 from opentracing import SpanContext as OpenTracingSpanContext
 
+from ddtrace.context import Context
+
 
 class SpanContext(OpenTracingSpanContext):
-    """"""
+    """Implementation of the opentracing span context."""
+
+    __slots__ = ['_context', '_baggage']
```

Comment:
See also no use of `__slots__` on the [base class](https://github.com/opentracing/opentracing-python/blob/2.0.0rc2/opentracing/span.py#L26)